props/prop_base.h

```diff
@@ -152,7 +162,13 @@ class PropBase : CommandParser, Looper, protected SaberBase {
       SetMute(false);
     }
   }
-
+	
+#ifdef DYNAMIC_CLASH_THRESHOLD
+  static float clash_threshold_;
+  static float GetCurrentClashThreshold() { return clash_threshold_; }
+  static void SetClashThreshold(float clash_threshold) { clash_threshold_ = clash_threshold; }
+#endif 
```

Comment:
In here, we could also add:

#undef CLASH_THRESHOLD_G
#define CLASH_THRESHOLD_G clash_threshold_

This can simplify some of the code below.

Reply:
added and remove ifdef in DoAccel()